requirements.txt

```diff
@@ -24,6 +24,8 @@ ruamel.yaml==0.18.5
 vcrpy-unittest==0.1.7
 user-agents==2.2.0
 dnspython==2.6.1
+werkzeug==2.3.8
```

Comment:
Why do we need this as separate dependency?

Reply:
@bartaz Without it we get the following error: `ImportError: cannot import name 'url_quote' from 'werkzeug.urls'`